RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -539,5 +539,6 @@ void main()
 			auto_log_error("Error running auto_pre_adv.ash, setting auto_interrupt=true");
 			set_property("auto_interrupt", true);
 		}
+		auto_interruptCheck();
```

Comment:
without this line, on a failure to do pre_ascension we will do 1 adv without proper preparations and then abort.
with this line we will properly shut down the script on the spot to prevent wasting that adventure.

this also solves the issue where we get an abort and then run it again and get a second abort. as what happened was:
1. pre adventure failed. it sets interrupt setting.
2. adventure done. adv fail somehow due to expecting some preparation that did not happen, such as equipping a piece of gear. this causes an immediate abort rather than a proper shutdown.
3. we run autoscend again, interrupt setting detected, proper shutdown is performed.